test/e2e/webhook_e2e_test.go

```diff
@@ -0,0 +1,955 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/stretchr/testify/require"
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/controller/install"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/operatorclient"
+)
+
+var _ = Describe("Webhook", func() {
+	It("creates global webhooks", func() {
```

Comment:
Instead of having the entire test in a single `It` spec, you could use ginkgo's container blocks to restructure the test. 
For example: 
```
Describe("Given a CSV containing a webhook in its spec", func() { 
      BeforeEach(func(){
          // creation of csv and webhook
      })
      AfterEach(func(){
         // optional cleanups if needed
      })
      It("creates the webhook", func() {
           // use gomega's matcher library to verify webhook creation
      })
})
```
This way, if you choose to reuse the same CSV for a different test, you can add that test as an `It` spec inside of this Describe block. That way, the `BeforeEach` does the setup required for you. 
This avoids code duplication and also gives better error descriptions(using the descriptions in container blocks) in case of a test failure